docs/docs/Deployment/deployment-enterprise-database.mdx

```diff
@@ -0,0 +1,271 @@
+---
```

Comment:
Could you move and rename this file to `/Develop/enterprise-database-guide.mdx` or something like that?

Eventually, I want to reorganize the directories to match our new IA.